app/components/gh-member-settings-form.js

```diff
@@ -33,12 +33,13 @@ export default Component.extend({
         let subscriptions = this.member.get('stripe');
         if (subscriptions && subscriptions.length > 0) {
             return subscriptions.map((subscription) => {
+                const status = subscription.status === 'past_due' ? 'Past due' : subscription.status;
                 return {
                     id: subscription.id,
                     customer: subscription.customer,
                     name: subscription.name || '',
                     email: subscription.email || '',
-                    status: subscription.status,
+                    status: status,
```

Comment:
IMO we should have an extra property that's like "human readable status" or "display status" - just so that we can use the original status values if we ever need to switch on them in the future - what do you think?

Reply:
Yeah agree, good point. I'll add a  `statusLabel` field with the human readable version so it can be easily extended 👍